### PR TITLE
Launch 2nd Smart Society Disengagement/Intervention Experiment on Galaxy Zoo

### PR DESCRIPTION
**Pull request by alexbfree**
This effectively reinserts the same code as #211, no differences to the code except naming.

### CODE REVIEW

**Comment by alexbfree** [comment]
can someone review & merge this please? maybe @parrish or @brian-c ? or any dev for that matter.
hoping to launch this today. Thanks in advance!

File: app/controllers/classify.coffee [hunk 1]
```diff
@@ -7,6 +7,11 @@ Favorite = require 'zooniverse/lib/models/favorite'
 User = require 'zooniverse/lib/models/user'
 UserGroup = require 'models/user_group'
 LoginForm = require 'zooniverse/lib/controllers/login_form'
+Intervention = require 'lib/intervention'
+Analytics = require 'lib/analytics'
+
+window.delay = (ms, fn)-> setTimeout(fn, ms)
+window.timer = (ms, fn)-> setInterval(fn, ms)
```
Comment (brian-c) [inline app/controllers/classify.coffee]: Booo globals.

**Comment by brian-c** [comment]
Looks good to me.

**Comment by alexbfree** [comment]
Thanks @brian-c !

**Pull request merged**